doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,247 @@
+.. _setup_logical_partition_scenario_on_nuc7i7dnb:
+
+Guidelines of Setup Logical Partition Scenario on NUC7i7DNB
+############################################################
+
+ACRN hypervisor supports logical partition scenario, in which the User OS (such
+as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and
+directly access isolated PCI devices. The following guidelines provide step by
+step instructions on how to set up the ACRN hypervisor logical partition
+scenario on NUC7i7DNB boards running two pre-launched VMs. The logical partition
+scenario on NUC7i7DNB is shown in
+:numref:`logical_partition_scenario_on_nuc7i7dnb`.
+
+.. figure:: images/logical_partition_scenario_on_nuc7i7dnb.png
+   :align: center
+   :width: 400px
+   :name: logical_partition_scenario_on_nuc7i7dnb
+
+   The logical partition scenario on NUC7i7DNB
+
+Prerequisites
+*************
+
+* `NUC7i7DNB <https://ark.intel.com/content/www/us/en/ark/products/130394/intel-nuc-board-nuc7i7dnbe.html>`_
```

Comment:
Let's list the NUC Kit models we support here instead of the board inside the NUC.

Reply:
Ok, list "Intel NUC Kit NUC7i7DNHE" here, thanks!

Reply:
Fix in next version